scipy/stats/tests/test_stats.py

```diff
@@ -477,6 +477,11 @@ def test_len1(self):
         y = [2]
         assert_raises(ValueError, stats.pearsonr, x, y)
 
+    def test_complex_data(self):
+        x = [-1j, -2j, -3.0j]
+        y = [-1j, -2j, -3.0j]
+        assert_raises(ValueError, stats.pearsonr, x, y)
```

Comment:
There are many `ValueErrors` out there. This test should only pass if the one we have in mind is raised.
```suggestion
        message = 'This function does not support complex data'
        with pytest.assert(ValueError, match=message):
            stats.pearsonr(x, y)
```